react-blocks/src/blocks/Splittwocolumns/Splittwocolumns.js

```diff
@@ -0,0 +1,282 @@
+import {Component,Fragment} from "@wordpress/element";
+import {
+  TextControl,
+  TextareaControl,
+  FocalPointPicker,
+  ServerSideRender,
+  Button,
+  SelectControl
+} from '@wordpress/components';
+import {BlockControls,MediaUpload,MediaUploadCheck} from "@wordpress/editor";
+
+import {Preview} from '../../components/Preview';
+
+export class Splittwocolumns extends Component {
+    constructor(props) {
+      super(props);
+    }
+
+    renderEdit() {
+      let tag_list   = this.props.tagsList.map((tag) => ({label: tag.name, value: tag.id}));
+      tag_list.unshift({label: '--Select Tag--', value: 0});
+
+      let issuepage_list = this.props.issuepageList.map((issue) => ({label: issue.title.raw, value: issue.id}));
+      issuepage_list.unshift({label: '--Select Issue--', value: 0});
+
+      const getIssueImageOrButton = (openEvent) => {
+        if ( this.props.issue_image ) {
+
+          return (
+
+            <img
+              src={ this.props.issue_image_url }
+              onClick={ openEvent }
+              className="splittwocolumns__issue_imgs"
+              width={'250px'}
+              style={{padding: '10px 10px'}}
+            />
+
+          );
+        }
+        else {
+          return (
+            <div className="button-container">
+              <Button
+                onClick={ openEvent }
+                className="button">
+                + {__('Select Image for Issue', 'p4ge')}
+              </Button>
+
+              <div>{__('(Optional)', 'p4ge')}</div>
+            </div>
+          );
+        }
+      };
+
+      const onCampaignImageFocalPointChange = (openEvent) => {
```

Comment:
same with getIssueImageOrButton, could you extract this to a class function to reduce the size of renderEdit()?